akka-persistence-typed/src/main/scala/akka/persistence/typed/javadsl/EventSourcedBehavior.scala

```diff
@@ -234,6 +242,12 @@ abstract class EventSourcedBehavior[Command, Event, State] private[akka] (
     scaladsl.EventSourcedBehavior.lastSequenceNumber(ctx.asScala)
   }
 
+  /**
+   * Override to define a custom stash capacity per entity.
+   * If not defined, the default `akka.persistence.typed.stash-capacity` will be used.
+   */
+  def withStashCapacity: Optional[java.lang.Integer] = Optional.empty()
```

Comment:
To align better with the naming of the other methods that can be overridden here I think we should name this `stashCapacity`.


scaladsl is different because there it's a "builder"